force-app/main/default/classes/DummyJSONCallout.cls

```
@@ -34,18 +34,32 @@ public with sharing class DummyJSONCallout {
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
 
-	public static void getDummyJSONUserFromId(String dummyUserId) {
+	@future(callout=true)
+	 public static void getDummyJSONUserFromId(String dummyUserId) {
 		// Create HTTP request to send.
-
+		HttpRequest req = new HttpRequest();
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
+		req.setEndpoint('callout:DummyJSON/users/' + dummyUserId);
+		req.setMethod('GET');
 
-		// Set the HTTP method to GET.
+		Http http = new Http();
 
 		// Send the HTTP request and get the response.
-
-		// If the HTTP response code is successful, parse the JSON response and update the contact.
-		Contact cont = null;
-		upsert cont DummyJSON_Id__c; //insert/update from the JSON response using the external id (dummyUserId)
+		try {
```

Avoid using Try/Catch if you are just going to debug. Try/Catch is meant to catch errors and gracefully recover or do something with that error. Just debugging can cause silent errors that never get addressed.